code/modules/clothing/spacesuits/alien_suits.dm

```diff
@@ -143,30 +143,48 @@
 	desc = "An almost organic looking nonhuman pressure suit."
 
 /obj/item/clothing/under/vox
-	has_sensor = FALSE
-	icon = 'icons/obj/clothing/species/vox/uniforms.dmi'
+	icon = 'icons/mob/clothing/species/vox/under/misc.dmi'
 	species_restricted = list("Vox")
 	sprite_sheets = list(
 		"Vox" = 'icons/mob/clothing/species/vox/under/misc.dmi'
 		)
 
 /obj/item/clothing/under/vox/vox_casual
-	name = "alien clothing"
+	name = "Vox jumpsuit"
 	desc = "This doesn't look very comfortable."
-	icon_state = "vox-casual-1"
-	item_color = "vox-casual-1"
-	item_state = "vox-casual-1"
+	icon_state = "vox-jumpsuit_s"
+	item_color = "vox-jumpsuit"
+	item_state = "vox-jumpsuit"
 	body_parts_covered = LEGS
```

Comment:
This should probably be removed, it looks like it covered all the default jumpsuit spots, which is `body_parts_covered = UPPER_TORSO|LOWER_TORSO|LEGS|ARMS`
```suggestion
```

Reply:
Fair enough. It's an artifact from the old version, and it makes sense with the sprite update.